src/routes/my-trips.js

```diff
@@ -1,30 +1,39 @@
-import * as utils from '../utils.js'
+import * as utils from '../utils.js';
 
 export function get(req, res) {
-  const userId = req.user
-  const _24_HOURS_IN_MS = 86400000
-  const now = new Date()
-
```

Comment:
looks like you ran a lint or something here? this needs to be removed from the diff (plus it fails the lint rules).